src/app/components/turnos/gestor-agendas/operaciones-agenda/revision-agenda.html

```diff
@@ -452,6 +452,9 @@ <h5>
                                           *ngIf="isRegistradoProfesional(turno) && !isAuditado(turno)">|
                                         Registrado por Profesional</span>
                                     <span [ngClass]="{'text-success': true}" *ngIf="isAuditado(turno)">| Auditado</span>
+                                    <br><small><span [ngClass]="{'text-info': true}"
```

Comment:
Si siempre va en true, directamente usar class
```suggestion
                                    <br>
                                        <small>
                                            <span class="text-info"
```

Reply:
corregido